docs/configuration.rst

```diff
@@ -1,16 +1,108 @@
++------------------+------------------------------+--------------+-----------------------------------------+
+| Schema Registry  | cp-schema-registry           | Open Source  | - confluent-schema-registry             |
++------------------+------------------------------+--------------+-----------------------------------------+
+| REST Proxy       | cp-kafka-rest                | Open Source  | - confluent-kafka-rest                  |
++------------------+------------------------------+--------------+-----------------------------------------+
+
+Note: The Kafka Connect image is labeled as "Enterprise" simply because it contains the Confluent Control Center package.  That package enables the deployed connectors to collect the metrics visualized in Confluent Control Center.   No explicit license is required when using the Kafka Connect image on its own.
+
+Configuration Notes
+-------------------
+
+*  Docker for Mac
+
+	We do not recommend using these images with Docker for Mac at this time.  The primary reason is that Docker for Mac does not update the local /etc/hosts file with the hostnames of the deployed containers.   This makes it difficult to access the containerized cluster with client applications running directly on the Mac.  Additionally, the semantics of ``--net=host`` are not clear, so deploying containers with host networking on Docker for Mac is not reliable.  More details on these issues can be found at:
+
+	- `Hostname Issue <https://forums.docker.com/t/docker-for-mac-does-not-add-docker-hostname-to-etc-hosts/8620/4>`_
+	- Host networking on Docker for Mac: `link 1 <https://forums.docker.com/t/should-docker-run-net-host-work/14215>`_, `link 2 <https://forums.docker.com/t/net-host-does-not-work/17378/7>`_, `link 3 <https://forums.docker.com/t/explain-networking-known-limitations-explain-host/15205/4>`_
+
+*  Persistent Data (Mounted Volumes)
+
+	When deploying the Kafka and ZooKeeper images, you should always use `mounted volumes <operations/external-volumes.html>`_ for the file systems those images use for their persistent data.  This ensures that the containers will retain their proper state when stopped and restarted.  The other images maintain their state directly in Kafka topics, so mounted volumes are not usually required for those containers.
+
+*  Bridge Networking vs. Host Networking
+
+	Bridge networking is currently only supported on a single host.  For multiple hosts, you will need to use overlay networks which are not currently supported. It order to expose Kafka to clients outside of the bridge network, you need to find the container IP and put it in ``advertised.listeners``.  This can be difficult to achieve depending on how you're using the images.  Furthermore, it can add a network hop and may not be as performant as the host network, which shares the network stack.  In summary, host networking is the recommended option in the following cases:
```

Comment:
It order -> In order

Reply:
better yet, simply `To expose...`. the conjunction "in order to" is generally unnecessary in tech docs and can be deleted in most cases.